flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/rank/window/WindowRankOperatorBuilder.java

```diff
@@ -116,11 +118,17 @@ public WindowRankOperatorBuilder windowEndIndex(int windowEndIndex) {
         return this;
     }
 
+    public WindowRankOperatorBuilder isEventTime(Boolean isEventTime) {
```

Comment:
nit: `isEventTime` looks like a predicate, could we just use `with` or `set` prefix for the set method?